mpy-cross/README.md

```diff
@@ -2,15 +2,28 @@ MicroPython cross compiler
 ==========================
 
 This directory contains the MicroPython cross compiler, which runs under any
-Unix-like system and compiles .py scripts into .mpy files.
+Windows and Unix-like system and compiles .py scripts into .mpy files.
 
-Build it as usual:
+Build it on unix-like systems as usual:
 
     $ make
 
-The compiler is called `mpy-cross`.  Invoke it as:
+To compile using Visual Studio 2013 (or higher):
+
+Open mpy-cross.vcxproj and build
+
+To compile using Visual Studio 2013 (or higher) commandline:
+
+    msbuild mpy-cross.vcxproj
+
+
+The compiler is called `mpy-cross`.  On unix-like invoke it as:
 
     $ ./mpy-cross foo.py
+    
+On windows invoke it as:
```

Comment:
This is basically the same as for the unix-like build so doesn't really need to be repeated imo